drivers/clock_control/clock_control_nrf2_hfxo.c

```diff
@@ -21,6 +21,7 @@ struct dev_data_hfxo {
 	onoff_notify_fn notify;
 	struct k_timer timer;
 	sys_snode_t hfxo_node;
+	uint16_t request_count;
```

Comment:
ZLI support is needed only if ZLI is enabled . Would be better to compile it only then. It adds ~100 bytes of code.

Reply:
Added IS_ENABLED(CONFIG_ZERO_LATENCY_IRQS) conditions. That caused a extraction of `start_hfxo`/`stop_hfxo` functions.